docs/testing/debug-desktop-using-edge-chromium.md

```diff
@@ -13,37 +13,23 @@ Office Add-ins running on Windows can use the Debugger for Microsoft Edge extens
 - [Visual Studio Code](https://code.visualstudio.com/) (must be run as an administrator)
 - [Node.js (version 10+)](https://nodejs.org/)
 - Windows 10
-- [Microsoft Edge Chromium available to Windows Insiders](https://www.microsoftedgeinsider.com/)
+- A combination of platfrom and Office host that supports Microsoft Edge with WebView2 (Chromium-based) as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). If your version of Microsoft 365 is earlier than 2101, you will need to install WebView2. Use the instructions for installing it at [Microsoft Edge WebView2 / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/).
 
 ## Install and use the debugger
 
 1. Create a project using the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office). You can use any one of our quick start guides, such as the [Outlook add-in quickstart](../quickstarts/outlook-quickstart.md), in order to do this.
 
 > [!TIP]
-> If you aren't using a Yeoman generator based add-in, you need to adjust a registry key. While in the root folder of your project, run the following in the command line.
+> If you aren't using a Yeoman generator based add-in, you may be prompted to adjust a registry key. While in the root folder of your project, run the following in the command line.
  `office-add-in-debugging start <your manifest path>`
```

Comment:
The formatting is off for this line on the review site. I think adding the > will help. 
```suggestion
 > `office-add-in-debugging start <your manifest path>`
```